workers/unity/Assets/Fps/Config/TileTypes/DefaultTiles.asset

```diff
@@ -12,12 +12,12 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: ca0a68eabc509694b9a201e0d90358ed, type: 3}
   m_Name: DefaultTiles
   m_EditorClassIdentifier: 
-  Tiles:
+  tiles:
   - {fileID: 1853307461053178640, guid: f4b0c5a26e0dde042af81ff54b6149bb, type: 3}
   - {fileID: 1853307461053178640, guid: 77073bec1cfdc74448a1433b0c2eb0a0, type: 3}
   - {fileID: 1853307461053178640, guid: d6da8e0d7a1d5654e8b23301a33af160, type: 3}
   - {fileID: 1853307461053178640, guid: 108e84c74f708804abfc1ac92731b83d, type: 3}
   - {fileID: 1853307461053178640, guid: 46057f07999102d459eb1e5a48c6347f, type: 3}
   - {fileID: 1853307461053178640, guid: 7c12f1fe6891d5747b06b5c2cc0a9650, type: 3}
-  ChanceOfEmptyTile: 0.05
-  DisplayColor: {r: 1, g: 1, b: 1, a: 0.5019608}
+  chanceOfEmptyTile: 0
```

Comment:
Is it correct that this chance has changed?

Reply:
oh, good spot. Not, it is not correct!